imports/users-presence/server/activeUsers.js

```diff
@@ -25,4 +26,12 @@ UserPresenceEvents.on('setUserStatus', (user, status/* , statusConnection*/) =>
 		STATUS_MAP[status],
 		statusText,
 	]);
+};
+
+settings.get('Troubleshoot_Disable_Presence_Broadcast', (key, value) => {
+	if (value) {
+		return UserPresenceEvents.removeListener('setUserStatus', setUserStatus);
+	}
+
+	UserPresenceEvents.on('setUserStatus', setUserStatus);
```

Comment:
due to the nature of our settings, on a multi instance environment this is being called every time a new instance is started, thus registering the callback multiple times.